packages/ember-runtime/lib/system/core_object.js

```diff
@@ -916,31 +949,31 @@ if (DEBUG) {
     @method _onLookup
   */
   ClassMixinProps._onLookup = injectedPropertyAssertion;
-  /**
-    Returns a hash of property names and container names that injected
-    properties will lookup on the container lazily.
+}
 
-    @method _lazyInjections
-    @return {Object} Hash of all lazy injected property keys to container names
-    @private
-  */
-  ClassMixinProps._lazyInjections = function() {
-    let injections = {};
-    let proto = this.proto();
-    let key;
-    let desc;
+/**
+  Returns a hash of property names and container names that injected
+  properties will lookup on the container lazily.
 
-    for (key in proto) {
-      desc = proto[key];
-      if (desc instanceof InjectedProperty) {
-        injections[key] = `${desc.type}:${desc.name || key}`;
-      }
+  @method _lazyInjections
+  @return {Object} Hash of all lazy injected property keys to container names
+  @private
+*/
+ClassMixinProps._lazyInjections = function() {
```

Comment:
This should be moved back into the `if (DEBUG) {` guard (it was changed in a PR recently, so it likely was missed during rebasing here).

Reply:
Ah yep, let me fix that up